CLAPACK/Makefile.envs

```diff
@@ -0,0 +1,2 @@
+CFLAGS_OPT=-O2
+LDFLAGS_OPT=-O2
```

Comment:
This is defined as a separate file so we can include it in the [CLAPACK-WA/Makefile](https://github.com/iodide-project/pyodide/pull/238/files#diff-1634a8032d156dab9781e40ecbb8db36R20) one level down, without having to change the patches. I think, make is expected to pass variables to child make commands, but because we use `emmake make` it doesn't seem to be the case (or otherwise I'm missing something).

Reply:
Seems fine as a workaround.